answers/README.md

```diff
@@ -0,0 +1,165 @@
+# ANSWERS
+
+| Method | Route                        | Description                                                    |
+| ------ | ---------------------------- | -------------------------------------------------------------- |
+| POST   | [/answers](#post---answers)  | it will be to submit answer by the peers                       |
+| PATCH  | [/answers](#patch---answers) | for updating the answers(will primarily be using for approval) |
+| GET    | [/answers](#get---answers)   | it will be to get the answers                                  |
+
+### POST - /answers
+
+It will be used for peers to answer questions.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - **`answer=[STRING]`** required - (The answer text)
+  - **`answeredBy=[STRING]`** required (The ID of the peer who answered the question.)
```

Comment:
- will this open up us to a vulnerability where if I get to know the ID of other participants, I can submit answers on their behalf?

Reply:
no because we're not taking id from any input it will be taken by 100ms client sdk

Reply:
- can I access other user's IDs make an API call (for example from Postman) and submit an answer on their behalf? (just putting this out there feel free to resolve this after reading)

Reply:
no it's will not possible